content/updates/migrate-off-tfc.md

```diff
@@ -0,0 +1,112 @@
+---
+visible: true
+draft: false
+title: "How to Migrate off Terraform Cloud"
+author: Veronika Gnilitska
+slug: how-to-migrate-off-tfc
+date: 2024-07-25
+description: "Need to to migrate off Terraform Cloud? We're happy to share some tips about preparation, pitfalls, and the process itself based on Masterpoint's experience."
+image: /img/updates/migrate-off-tfc.jpeg
+callout: <p>👋 <b>If you're ready to take your infrastructure to the next level, we're here to help. We love to work together with engineering teams to help them build well-documented, scalable, automated IaC that make their jobs easier. <a href='/contact'>Get in touch!</a>
+---
+
+# Introduction
+
+Migrating off Terraform Cloud could seem a dreaded task, but with proper planning and execution, it can be done smoothly and efficiently. Whether you're moving due to cost considerations, the need for more control, or other reasons, this post will walk you through the essential steps and action items to successfully transition your infrastructure management away from Terraform Cloud.
```

Comment:
I suggest we shorted to TFC. Thoughts?

```suggestion
Migrating off Terraform Cloud (TFC) could seem a dreaded task, but with proper planning and execution, it can be done smoothly and efficiently. Whether you're moving due to cost considerations, the need for more control, or licensing considerations, this post will walk you through the essential steps and action items to successfully transition your infrastructure management away from TFC.
```

Reply:
Hm, considering [this](https://developer.hashicorp.com/terraform/cloud-docs):
>Effective April 22, 2024, Terraform Cloud is now HCP Terraform. 

Should we be consistent with the updated naming and change Terraform Cloud to HCP Terraform?

Reply:
`licensing considerations` suggestion added to the doc